src/Dockerfile

```diff
@@ -3,8 +3,6 @@ ENV USER_UID=1001 USER_NAME=opslevel
 ENTRYPOINT ["/usr/local/bin/opslevel"]
 WORKDIR /app
 RUN apt-get update && \
-    apt-get install -y curl && \
-    apt-get purge && apt-get clean && apt-get autoclean && \
-    curl -o /usr/local/bin/jq http://stedolan.github.io/jq/download/linux64/jq && \
-    chmod +x /usr/local/bin/jq
+    apt-get install -y curl jq && \
+    apt-get purge && apt-get clean && apt-get autoclean
```

Comment:
running the following locally i get jq 1.6 not 1.7. We probably need to update the base ubuntu image so we pull in a newer package? 🤷 (or did jq 1.7 not get published)

```
docker pull ubuntu:focal
docker run -it --rm ubuntu:focal /bin/bash
apt-get update
apt-get install -y curl jq
jq --version
jq-1.6
```

Reply:
@rocktavious your output is correct. This was an oversight. 

```
Package jq
[focal (20.04LTS)](https://packages.ubuntu.com/focal/jq) (utils): lightweight and flexible command-line JSON processor [universe]
1.6-1: amd64 arm64 armhf i386 ppc64el riscv64 s390x
[focal-updates](https://packages.ubuntu.com/focal-updates/jq) (utils): lightweight and flexible command-line JSON processor [universe]
1.6-1ubuntu0.20.04.1: amd64 arm64 armhf i386 ppc64el riscv64 s390x
[jammy (22.04LTS)](https://packages.ubuntu.com/jammy/jq) (utils): lightweight and flexible command-line JSON processor
1.6-2.1ubuntu3: amd64 arm64 armhf i386 ppc64el riscv64 s390x
[lunar (23.04)](https://packages.ubuntu.com/lunar/jq) (utils): lightweight and flexible command-line JSON processor
1.6-2.1ubuntu3: amd64 arm64 armhf i386 ppc64el riscv64 s390x
[mantic (23.10)](https://packages.ubuntu.com/mantic/jq) (utils): lightweight and flexible command-line JSON processor
1.6-3: amd64 arm64 armhf i386 ppc64el riscv64 s390x
[noble](https://packages.ubuntu.com/noble/jq) (utils): lightweight and flexible command-line JSON processor
1.7.1-2: amd64 arm64 armhf i386 ppc64el riscv64 s390x
```

Source: https://packages.ubuntu.com/search?keywords=jq

I just pushed a change to make this PR use `ubuntu:noble` instead. I tested it on my machine and `jq --version` is 1.7.

Reply:
awesome!